ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/424_model_serving_bias_metrics.robot

```diff
@@ -0,0 +1,188 @@
+*** Settings ***
+Documentation       TestSuite testing Bias metrics for Deployed Models
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/OCP.resource
+Resource            ../../../Resources/Common.robot
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Resource            ../../../Resources/Page/Operators/ISVs.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource
+Resource            ../../../Resources/Page/OCPDashboard/Monitoring/Metrics.robot
+Library             OpenShiftLibrary
+Suite Setup         Bias Metrics Suite Setup
+Suite Teardown      Bias Metrics Suite Teardown
+
+*** Variables ***
+${DEFAULT_MONITORING_NS}=            openshift-user-workload-monitoring
+${PRJ_TITLE}=                        model-serving-project
+${PRJ_TITLE1}=                       model-project-test
+${MODEL_ALPHA}=                      demo-loan-nn-onnx-alpha
+${MODEL_PATH_ALPHA}=                 trusty/loan_model_alpha.onnx
+${MODEL_BETA}=                       demo-loan-nn-onnx-beta
+${MODEL_PATH_BETA}=                  trusty/loan_model_beta.onnx
+${TRUSTYAI_RESOURCEPATH}=            ods_ci/tests/Resources/Files/TrustyAI
+${TRUSTYAI_CR_FILEPATH}=             ${TRUSTYAI_RESOURCEPATH}/trustyai_crd.yaml
+${MONITORING_CONFIG_FILEPATH}=       ods_ci/tests/Resources/Files/uwm_cm_conf.yaml
+${UWM_CONFIG_FILEPATH}=              ods_ci/tests/Resources/Files/uwm_cm_enable.yaml
+${aws_bucket}=                       rhods-public
+${RUNTIME_NAME}=                     Model Bias Serving Test
+${PRJ_DESCRIPTION}=                  Model Bias Project Description
+${PRJ_DESCRIPTION1}=                 Model Bias Project Description 1
+${framework_onnx}=                   onnx - 1
+
+*** Test Cases ***
+Verify DIR Bias Metrics Available In CLI For Models Deployed Prior To Enabling Trusty Service For Admin User
+    [Documentation]    Verifies that the Bias metrics are available in Metrics Console for a model which was
+    ...                 deployed prior to enabling the TrustyAI service
+    [Tags]    Smoke
```

Comment:
As both tests seem to do similar things, consider adding one to Smoke and another to Sanity in order to speed up Smoke execution

Reply:
Done